### PR TITLE
fix: actions configuration

### PR DESCRIPTION
**Pull request by alnr**
The `flows.registration.after.hooks` section is used only if the `flows.registration.after.{password,oidc}` sections are empty.

This was confusing to users, because by default we have the `session` hook in this section, which would then case the webhooks in the "global" section to not be sent.

### CODE REVIEW

File: docs/guides/integrate-with-ory-cloud-through-webhooks.mdx [hunk 1]
```diff
@@ -79,23 +79,23 @@ selfservice:
...
-            config:
-              url: https://webhook.traget/example
-              method: POST
-              body: base64://ey...
-              response:
-                ignore: false
-                parse: false
-              auth:
-                # ...
+        hooks: [] # These acions are executed only if the `password` and `oidc` sections below are empty
```
[the remark this answers is not in the document]
Comment (aeneasr) [reply docs/guides/integrate-with-ory-cloud-through-webhooks.mdx:82]: Yup: https://github.com/ory/kratos/blob/3d07161ab4a7e19ebb786c9983ace6e2d9617c1e/driver/registry_default_registration.go#L37-L40

[the remark this answers is not in the document]
Comment (alnr) [reply docs/guides/integrate-with-ory-cloud-through-webhooks.mdx]: rephrased

File: docs/guides/integrate-with-ory-cloud-through-webhooks.mdx [hunk 2]
```diff
@@ -79,23 +79,23 @@ selfservice:
...
-                ignore: false
-                parse: false
-              auth:
-                # ...
+        hooks: [] # These acions are executed only if no more specific hooks in the `password` or `oidc` sections are defined
         password:
-          after:
-            # Actions in this sections will be executed after sign up (or login, settings) with password
-            # ...
+          # Actions in this sections will be executed after sign up (or login, settings) with password
```
Comment (tomekpapiernik) [inline docs/guides/integrate-with-ory-cloud-through-webhooks.mdx:84]:
```suggestion
          # Actions in this section will be executed after registration (or login, settings) with a password
```

File: docs/guides/integrate-with-ory-cloud-through-webhooks.mdx [hunk 3]
```diff
@@ -79,23 +79,23 @@ selfservice:
...
+                response:
+                  ignore: false
+                  parse: false
+                auth:
+                  # ...
+            - hook: session # see https://www.ory.sh/docs/actions/session
         oidc:
-          after:
+          hooks:
             # Actions in this section will be executed after sign up (or login, settings) with a social sign-in provider / OpenID Connect
```
Comment (tomekpapiernik) [inline docs/guides/integrate-with-ory-cloud-through-webhooks.mdx:99]:
```suggestion
            # Actions in this section will be executed after registration (or login, settings) with a social sign-in provider / OpenID Connect
```

**Review by aeneasr — APPROVED** [review]

**Pull request merged**